user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleContentComponent.java

```diff
@@ -85,6 +102,56 @@ private void reloadOnBatchRegistration() {
         event -> displayComponentInContent(sampleDetailsComponent));
   }
 
+  private void addBatchUpdateListeners() {
+    batchDetailsComponent.addBatchCreationListener(this::registerBatch);
+    batchDetailsComponent.addBatchDeletionListener(this::openBatchDeletionConfirmation);
+    batchDetailsComponent.addBatchEditListener(this::editBatch);
+    batchDetailsComponent.addBatchViewListener(this::viewBatch);
+  }
+
+  private void registerBatch(CreateBatchEvent createBatchEvent) {
+    onRegisterButtonClicked();
+  }
+
+  //Todo Determine how samples should be loaded and shown within batch
```

Comment:
I think you can delete this todo. It becomes clear when we work on that.

Reply:
done